plasmapy/formulary/lengths.py

```diff
@@ -101,18 +106,19 @@ def Debye_length(T_e: u.K, n_e: u.m**-3) -> u.m:
 )
 @particle_input(any_of={"charged", "uncharged"})
 def gyroradius(  # noqa: C901
-    B: u.T,
+    B: u.Quantity[u.T],
     particle: ParticleLike,
     *,
-    Vperp: u.m / u.s = np.nan * u.m / u.s,
-    T: u.K = None,
+    Vperp: u.Quantity[u.m / u.s] = np.nan * u.m / u.s,
+    T: u.Quantity[u.K] = None,
```

Comment:
Excited that these `Quantity` annotations are working!  Now these can be true type hints.

Reply:
Will these have to be altered in all other modules?

Reply:
Yep!  This is a first try to see if the new capability of `@validate_quantities` to deal with `Quantity[Unit]` style is working in action.  I'm hoping to figure out a way to automate this...which might end up meaning I'll need to delve into regular expressions.